1-mandatory/3-slideshow/style.css

```diff
@@ -1 +1,32 @@
 /** Write your CSS in here **/
+body {
+  background-color: aquamarine;
+}
+#slideshow {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  margin-top: 50px;
+  margin-left: 20px;
+}
+.buttons {
+  margin-left: 430px;
+}
+.slide-btn {
+  margin: 10px 10px;
+  color: rgb(18, 18, 98);
+  background-color: rgb(0, 166, 255);
+  font-size: medium;
+  font-weight: bold;
+  border-radius: 5px;
+}
+.slide-btn:hover {
+  background: rgb(39, 203, 184);
+}
+.slideImg {
+  width: 400px;
+}
+.delay-set {
+  margin-left: 100px;
```

Comment:
Try to avoid hardcoding margins like this; it tends to break down when the page width changes. Think of things in terms of centering or being aligned left or right, relative to the window.